syntax = "proto3";
option java_package = "proto";
import "PriceEntity.proto";

message PricesStreamedFilePayload {
     PriceEntity price = 1;
}